src/curve.rs

```diff
@@ -231,6 +231,9 @@ pub trait Identity {
     /// Returns the identity element of the curve.
     /// Can be used as a constructor.
     fn identity() -> Self;
+
```

Comment:
I made a new `IsIdentity` trait in 911cce093, and then made a generic implementation which will exist for all point representations which have `Identity` and `CTEq` traits implemented.